Q7.py

```diff
@@ -0,0 +1,16 @@
+import re
+
+def valid_email(emails):
+    #I have make pattern variable, on bases of given condition using regex. 
+    pattern = r'\b[A-Za-z0-9._-]+@[A-Za-z0-9]+\.[A-Za-z]{2,3}\b'
+    #I have loop over the emails_list, use match and remove function to remove the unsupport mail form list.   
+    for i in emails[:]:
```

Comment:
Change name of `i` to email. Also no need of writing emails[:], you can directly loop through it `for email in emails`